chains/evm/contracts/pools/TokenPool.sol

```diff
@@ -245,20 +198,29 @@ abstract contract TokenPool is IPoolV2, Ownable2StepMsgSender {
     return i_rmnProxy;
   }
 
-  /// @notice Gets the pool's Router
-  /// @return router The pool's Router
-  function getDynamicConfig() public view virtual returns (address router, uint256 thresholdAmountForAdditionalCCVs) {
-    return (address(s_router), s_thresholdAmountForAdditionalCCVs);
+  /// @notice Gets the pools dynamic configuration.
+  function getDynamicConfig()
```

Comment:
A lot was added to the dynamic config which had their own functions before this PR. This saves meaningful contract space